projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-get-site-endpoint.php

```diff
@@ -271,6 +272,7 @@ class WPCOM_JSON_API_GET_Site_Endpoint extends WPCOM_JSON_API_Endpoint {
 		'blogging_prompts_settings',
 		'wpcom_production_blog_id',
 		'wpcom_staging_blog_ids',
+		'is_commercial',
```

Comment:
The option is stored in WPCOM, so we need it to be added / overridden by WPCOM here.